cypress/e2e/07-action-voter-registration.cy.ts

```diff
@@ -0,0 +1,159 @@
+/// <reference types="cypress" />
+
+import { mockRandomUser, mockWallet } from 'cypress/fixture/mocks'
+
+describe('action - voter registration without signin', () => {
+  it('action - voter registration - Navigate back and forth', () => {
```

Comment:
This prefix on the test describer is not necessary, the test runner already scopes all this runs under the `describe` above, and the idea of the `it` function is to describe how a feature should work
```suggestion
  it('should allow back and forth navigation', () => {
```

Reply:
you're right, thanks man